packages/terra-profile-image/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ## 3.41.1 - (January 4, 2022)
 
+* Changed 
+  * Minor dependency version bump
```

Comment:
Change this entry to something meaningful wrt the changes and move it to the unreleased section.